app/containers/HomeContainer/tests/index.test.js

```diff
@@ -54,4 +60,41 @@ describe('<HomeContainer /> tests', () => {
     await timeout(500);
     expect(submitSpy).toBeCalled();
```

Comment:
please make this toBeCalledWith and pass the params that is should be passed with

Reply:
Updated accordingly